_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+            <label class="form-label" for="select_01_suc">Default select menu with success</label>
+            <select class="form-input input-select has-success" name="select_01_suc" id="select_01_suc">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+
+        <div class="field">
+            <label class="form-label" for="select_02">Enhanced select menu</label>
+            <select class="form-input input-select replace-select" name="select_02" id="select_02">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="select_02_err">Enhanced select menu with error</label>
+            <select class="form-input input-select replace-select has-error" name="select_02_err" id="select_02_err">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="form-label" for="select_02_suc">Enhanced select menu with success</label>
+            <select class="form-input input-select replace-select has-success" name="select_02_suc" id="select_02_suc">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Radios</h4>
+<div class="example-set">
+    <fieldset class="field-group form-group">
```

Comment:
Thanks for the information. This class should be placed on an immediate ``<div>`` or containing element for the group of checkboxes/radio elements, not on the entire fieldset. I can see cases where a fieldset or ``.field-group`` would be comprised of mixed inputs. See the example markup below:

```html
<fieldset class="form-group">
    <legend class="form-group-hd sr-only">Your Personal Details</legend>
    
    <div class="field">
        <label class="form-label field-required" for="full_name_default">Your name (default text field)</label>
        <input class="form-input input-text" type="text" name="full_name_default" id="full_name_default" placeholder="e.g. Jane Doe" required>
    </div>

    <div class="field">
        <label class="form-label" for="gender">Your Gender</label>
        <select class="form-input input-select replace-select" name="gender" id="gender">
            <option value="">--</option>
            <option value="Male">Male</option>
            <option value="Female">Female</option>
            <option value="Other">Other</option>
        </select>
    </div>
    
    <div class="field-group">
        <h3 class="field-group-hd">What do you eat?</h3>

        <div class="field">
            <label class="form-label label-inline" for="food-preference-01">
                <input type="radio" class="form-input input-radio replace-radio" name="food-preference" id="food-preference-01" value="Omnivore">
                <span class="form-input-label">Omnivore</span>
            </label>
        </div>
         <div class="field">
            <label class="form-label label-inline" for="food-preference-02">
                <input type="radio" class="form-input input-radio replace-radio" name="food-preference" id="food-preference-02" value="Vegetarian">
                <span class="form-input-label">Vegetarian</span>
            </label>
        </div>
         <div class="field">
            <label class="form-label label-inline" for="food-preference-03">
                <input type="radio" class="form-input input-radio replace-radio" name="food-preference" id="food-preference-03" value="Vegan">
                <span class="form-input-label">Vegan</span>
            </label>
        </div>
    </div>

    <div class="field-group">
        <h3 class="field-group-hd">Where do you stand on politics?</h3>

        <div class="field">
            <label class="form-label label-inline" for="political-preference-01">
                <input type="checkbox" class="form-input input-radio replace-radio" name="political-preference" id="political-preference-01" value="Democrat">
                <span class="form-input-label">Democrat</span>
            </label>
        </div>
         <div class="field">
            <label class="form-label label-inline" for="food-preference-02">
                <input type="checkbox" class="form-input input-radio replace-radio" name="political-preference" id="political-preference-02" value="Republican">
                <span class="form-input-label">Republican</span>
            </label>
        </div>
         <div class="field">
            <label class="form-label label-inline" for="political-preference-03">
                <input type="checkbox" class="form-input input-radio replace-radio" name="political-preference" id="political-preference-03" value="Independent">
                <span class="form-input-label">Independent</span>
            </label>
        </div>
    </div>    
</fieldset>
```

In this case, I'd recommend we:

* change ``form-group`` to be used for grouping parts of a form
* change and use ``field-group`` to be used when a "field" is actually comprised of many fields - radio/checkboxes are the most common for this
* create a ``.field-group-hd`` element and class that can act as a header for the newly defined ``form-group`` elements